tmt/utils.py

```diff
@@ -27,6 +27,12 @@
 from collections import Counter, OrderedDict
 from collections.abc import Iterable, Iterator, Sequence
 from contextlib import suppress
+
+if sys.version_info < (3, 10):
+    from importlib_metadata import entry_points
+else:
+    from importlib.metadata import entry_points
+from importlib.readers import MultiplexedPath
```

Comment:
`mypy` is also complaining about this `MultiplexedPath`, but if you actually run it, it imports and works fine. The `__init__.py` in `importlib` is fairly complicated so it might be prone to errors.

Otherwise here are the links of how it navigates:
- https://github.com/python/cpython/blob/3.12/Lib/importlib/readers.py
- https://github.com/python/cpython/blob/3.12/Lib/importlib/resources/readers.py#L56